specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2894,17 +2890,32 @@
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
         },
-        "rebootPending": {
-          "type": "boolean",
-          "description": "Specifies whether this compute instance is pending for reboot to finish os patching."
-        },
+        "osPatchingStatus": {
+          "readOnly": true,
+          "description": "Metadata about the os patching.",
+          "$ref": "#/definitions/osPatchingStatus"
+        }
+      }
+    },
+    "osPatchingStatus": {
```

Comment:
Should be capital first letter compared to other properties here?OSPatchingStatus seems to diverge with its own block compared to the initial spec, but that change seems good to me and might be more clear.

Reply:
will update to upper case later.